jdaviz/core/launcher.py

```diff
@@ -0,0 +1,52 @@
+import ipyvuetify as v
+from ipywidgets import jslink
+from traitlets import Dict
+
+from jdaviz import configs as jdaviz_configs
+from jdaviz.core.data_formats import open as jdaviz_open
+
+
+def show_launcher(configs=['imviz', 'specviz', 'mosviz', 'cubeviz', 'specviz2d']):
```

Comment:
`configs` should be immutable, right?
```suggestion
def show_launcher(configs=('imviz', 'specviz', 'mosviz', 'cubeviz', 'specviz2d')):
```

Reply:
I'd like to avoid making `configs` immutable to try and support external configs (e.g. `lcviz`) one day. I'm not sure how we would, but I'm trying to keep things as flexible as possible